apps/marketing/src/components/(marketing)/widget.tsx

```diff
@@ -392,10 +393,11 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
           </DialogHeader>
 
           <DialogDescription>
-            By signing you signal your support of Documenso's mission in a <br></br>
-            <strong>non-legally binding, but heartfelt way</strong>. <br></br>
-            <br></br>You also unlock the option to purchase the early supporter plan including
-            everything we build this year for fixed price.
+            By signing you signal your support of Documenso's mission in a <br />
+            <strong>non-legally binding, but heartfelt way</strong>. <br />
+            <br />
+            You also unlock the option to purchase the early supporter plan including everything we
+            build this year for fixed price.
```

Comment:
Can we rephrase this line into something that is much friendlier? 😶  

Eg: 
`Join the exclusive club! Choose the early supporter plan to snag all our creations this year at a fixed, pocket-friendly rate.` 
or
`Your key to a year of excitement: the early supporter plan! Grab all our new releases at a fantastic, fixed price`

Reply:
@catalinpit @dguyen Would love to know your opinion on this

Reply:
I think the current is fine with maybe a few adjustments if we really want

Reply:
For the time being, I think it would be best to maintain the current line. We can discuss the changes in a separate pull request.